src/NuGetGallery.Core/GitHub/KiloFormatExtension.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Text;
+
+namespace NuGetGallery.GitHub
+{
+    public class KiloFormatExtension
```

Comment:
Should we put this into a generic place like `NuGetGallery.Core/Extensions`? We could rename this to something like `IntFormatExtensions`, and make `ToKiloFormat` an extension method on int. This would let us reuse this code in the future like `123.ToKiloFormat()`.